cmd/access/node_builder/access_node_builder.go

```diff
@@ -722,6 +723,17 @@ func (builder *FlowAccessNodeBuilder) BuildExecutionSyncComponents() *FlowAccess
 					if err != nil {
 						return nil, fmt.Errorf("could not load checkpoint file: %w", err)
 					}
+
+					// this is only supported when bootstrapping a new register db, otherwise the
+					// register db will throw exceptions when indexing an already indexed height.
+					if builder.forceResetIndexerHeight {
```

Comment:
Maybe better to use a height, so that the operator doesn't need to change it back to false after the reindex.
```suggestion
					shouldResetIndexHeight := builder.forceResetIndexerHeight == checkpointHeight 
					if shouldResetIndexHeight {
```

Reply:
this would require them to set the right height though. As is, this logic is only executed if the registers db is not bootstrapped, so it should never be called again on subsequent reboots.